doc/user-docs/UsingJsonSettings.md

```diff
@@ -107,6 +107,21 @@ add the following to your keybindings:
 This will _unbind_ <kbd>Ctrl+Shift+6</kbd>, allowing vim to use the keystroke
 instead of the terminal.
 
+### Binding multiple keys
+
+You can have multiple key chords bound to the same action, but they should each be in their own bindings. For example:
```

Comment:
"they should each be in their own bindings".... Can you restate the problem/solution without also using the word "binding" again? Like.... "they should each be defined in a separate block"